examples/videocomposer/configs/train.yaml

```diff
@@ -16,4 +16,4 @@ epochs: 50
 
 # accelerate or memory reduction
 dataset_sink_mode: True
-use_recompute: True
+use_recompute: False
```

Comment:
Why not using recompute by default?

Reply:
It scarifices speed for more memory. If 910B memory is enough, than it's not necessary by default.